documentation/src/docs/getting-started.md

```diff
@@ -89,7 +88,7 @@ _Example HelloWorldWriter output_
 Writing message: 'hello world' with routing-key: 'helloRoutingKey' to stream 'examples / helloStream'
 ...
 ```
-See the [README.md](https://github.com/pravega/pravega-samples/blob/v0.3.2/pravega-client-examples/README.md) file in the standalone-examples for more details
+See the [README](https://github.com/pravega/pravega-samples/blob/v0.3.2/pravega-client-examples/README.md) file in the standalone-examples for more details
     on running the HelloWorldWriter with different parameters.
 
 **Run the sample "HelloWorldReader"**
```

Comment:
@RaulGracia are these instructions up to date?

Reply:
yes, updated

Reply:
Yes, they are up to date and can be used to run the samples.